codalab/worker/bundle_manager.py

```diff
@@ -421,6 +420,22 @@ def _compute_request_memory(self, bundle):
             return formatting.parse_size(bundle.metadata.request_memory)
         return self._default_request_memory
 
+    def _compute_request_disk(self, bundle):
+        """
+        Compute the disk limit used for scheduling the run.
+        """
+        if bundle.metadata.request_disk:
+            return formatting.parse_size(bundle.metadata.request_disk)
+        return self._default_request_disk
```

Comment:
Double check that if this returns None (when self._default_request_disk is returned) then whatever is using the result of this function call doesn't crash

Reply:
This should not return `None`. We should always have a default request disk to limit how much disk can be used by any run. If we do not want to impose an arbitrary default request disk, I can make changes so it defaults to user disk quota left.  Thoughts? @percyliang @fabeschan